src/index.js

```diff
@@ -249,7 +262,7 @@ export const componentClasses = [
 const init = () => {
   if (!settings.disableAutoInit) {
     initCheckbox();
-    componentClasses.forEach((Clz) => {
+    componentClasses.forEach(Clz => {
```

Comment:
The change here may cause a lint error. Please note that the vanilla codebase hasn’t moved to Prettier (yet).